mms/tests/unit_tests/test_validate_messages.py

```diff
@@ -15,25 +15,35 @@
 
 
 def test_validate_load_message_missing_model_path():
-    invalid_object = {'command': 'some-command', 'modelName': 'some-model-name'}
+    invalid_object = {'command': 'some-command', 'modelName': 'some-model-name', 'handler': 'some handler string'}
     with pytest.raises(MMSError) as error:
         ModelWorkerMessageValidators.validate_load_message(invalid_object)
 
     assert error.value.get_code() == ModelServerErrorCodes.INVALID_LOAD_MESSAGE, 'error codes don\'t match'
-    assert error.value.get_message() == 'Load command missing \"model-path\" key'
+    assert error.value.get_message() == 'Load command missing \"modelPath\" key'
```

Comment:
Double qoute string to avoid using escape sequences
Eg:
  "Hello I'm here"
Double quote strings, is a better way always

Reply:
Done in eaa62a366e264cec1e8290a247a17f279b31fa63